.cruft.json

```diff
@@ -0,0 +1,20 @@
+{
+  "template": "git@github.com:NCI-GDC/bio-python-template.git",
+  "commit": "2538d116b1c2f3cf68b734c5a5179a4fc70acc9f",
+  "context": {
+    "cookiecutter": {
+      "full_name": "Charles Czysz",
+      "email": "czysz@uchicago.edu",
+      "github_username": "czyszCTDS",
+      "project_name": "aliquot-maf-tools",
+      "package_name": "aliquotmaf",
+      "project_short_description": "Tools for creating and filtering aliquot-level MAFs",
+      "min_python_version": "3.8",
+      "github_repo": "https://github.com/NCI-GDC/aliquot-maf-tools",
+      "commit": "3e2a935acaac77407e66d747480e45ded0ab3f98",
+      "_template": "git@github.com:NCI-GDC/bio-python-template.git"
```

Comment:
Using `cruft check` and `cruft update` to automatically apply updates from the template repo to this (and other) repos